cmake.eclass sets CMAKE_TOOLCHAIN_FILE to a temporary file, but
Qt expects it to not be temporary. Projects shouldn't be loading
gentoo's toolchain file outside ebuilds either way.

https://bugs.gentoo.org/984102
--- a/cmake/qt.toolchain.cmake.in
+++ b/cmake/qt.toolchain.cmake.in
@@ -23 +22,0 @@
-@init_original_toolchain_file@
